src/context.ts

```diff
@@ -75,3 +77,59 @@ export class OperonContextImpl implements OperonContext {
     return value as T;
   }
 }
+
+
+/**
+ * TODO : move logger and application, getConfig to a BaseContext which is at the root of all contexts
+ */
+export class InitContext {
+  
+  readonly logger: Logger ;
```

Comment:
```suggestion
  readonly logger: Logger;
```

This apply to other places, please 2x check diff before committing